src/main/resources/static/common/js/category.js

```diff
@@ -132,21 +132,18 @@ app.controller('categoryController', function($rootScope, $http, $location, $fil
         category.showCategoryRequisitionDetail = true;
 
         //Get cartItem by item id
-        $http({
-            method: 'GET',
-            url: '/proxy/v1/cartItems/' + itemId
-        }).then(function(result) {
-            var cartItem = result.data.data;
+        let getCartItemByIdSuccess = (cartItem) => {
             category.currentItem.inStock = cartItem.realInStock;
             category.currentItem.inRequisition = cartItem.quantity;
-            checkInventory(cartItem.realInStock,itemId,cartItem.quantity);
-            $interval(function(){category.loadingAnimation = false;category.showQuantity = true;},500,1);
-        }).catch(function(result) {
+            checkInventory(cartItem.realInStock, itemId, cartItem.quantity);
+            $interval(function() {category.loadingAnimation = false;category.showQuantity = true;}, 500, 1);
+        }
+        let getCartItemByIdError = (data) => {
```

Comment:
I think there is no need to call `GET /proxy/v1/assets/items/{itemId}` api in this method. Maybe it's a bad implementation when we wrote this code. We can have a discussion at meeting.

Reply:
I think so. Because the error from second call are the same with the first layer.